extensions/wikia/Oasis/Oasis.i18n.php

```diff
@@ -4936,52 +4944,52 @@
 	'oasis-navigation-v2-add-video' => '添加視頻',
 	'oasis-navigation-v2-edit-this-menu' => '編輯維基導航',
 	'oasis-navigation-v2-validation-next' => '下一頁',
-	'oasis-navigation-v2-new-photos' => '圖片',
+	'oasis-navigation-v2-new-photos' => '照片',
 	'oasis-on-the-wiki' => '本維基',
 	'oasis-desc' => '提供一種簡單的方式在Oasis皮膚介面上添加文檔',
 	'oasis-global-page-header' => 'Fandom',
 	'oasis-share' => '分享',
 	'oasis-button-random-page-tooltip' => '{{int:oasis-button-random-page}}',
 	'oasis-button-wiki-activity-tooltip' => '{{int:oasis-button-wiki-activity}}',
-	'oasis-global-nav-header' => 'Fandom導航',
-	'oasis-global-nav-create-wiki-alt' => '成立一個新的wiki',
-	'oasis-badge-notification' => '您剛剛赢得了"$2"徽章$3',
-	'oasis-badge-notification-see-more' => '快來看看您還可能赢得的其他徽章吧！',
-	'oasis-edits-counter' => '$1 <sup>{{PLURAL:$1|個編輯|個編輯}}已完成</sup>',
+	'oasis-global-nav-header' => '維基導航',
+	'oasis-global-nav-create-wiki-alt' => '創建一個維基',
+	'oasis-badge-notification' => '你剛剛贏了"$2"徽章$3',
+	'oasis-badge-notification-see-more' => '趕快來看看您可以贏得的更多徽章！',
+	'oasis-edits-counter' => '$1<sup>{{PLURAL:$1|次編輯|次編輯}}已完成</sup>',
 	'oasis-button-edit-wikia-local-menu' => '編輯本地社區功能表',
 	'oasis-button-edit-wikia-global-menu' => '編輯全域Fandom功能表',
 	'oasis-community-social-twitter-link' => 'https://twitter.com/getfandom',
 	'oasis-community-social-facebook-link' => 'https://www.facebook.com/getfandom',
 	'oasis-community-social-googleplus-link' => 'http://gplus.to/wikia',
 	'oasis-generic-error' => '我們很抱歉，處理您的請求時出錯。請稍後再試。',
-	'oasis-comments-showing-most-recent' => '顯示$1個最近的',
-	'oasis-comments-user-prompt' => '以$1的身份登入',
-	'oasis-comments-anonymous-prompt' => '匿名用户<br /> <a href="#" class="ajaxLogin">要登入？</a>',
+	'oasis-comments-showing-most-recent' => '顯示最近$1個',
+	'oasis-comments-user-prompt' => '以$1身份登錄',
+	'oasis-comments-anonymous-prompt' => '匿名使用者<br /> <a href="#" class="ajaxLogin">進行登錄？</a>',
```

Comment:
It seems in this case version in app repo was never updated - it should use wikitext to link to login page